Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,42 @@
 ﻿using System;
+using System.Threading;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
         {
+            var file = "./logger.txt";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'const' modifier to 'file'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5141367)